utils/dump_unique_sentences.py

```diff
@@ -0,0 +1,54 @@
+import csv
+import os
+# https://github.com/facebookresearch/EmpatheticDialogues/issues/21
+# https://github.com/facebookresearch/EmpatheticDialogues/issues/29
+# https://github.com/facebookresearch/EmpatheticDialogues/blob/master/empchat/datasets/empchat.py
+
+"""
+with open('empatheticdialogues/test.csv', newline='') as csvfile:
```

Comment:
@kohlisimranjit Can we use the configuration parameters for the path of "test.csv"?